filter/cond/README.md

```diff
@@ -0,0 +1,34 @@
+gogstash cond filter module
+=============================
+
+Condition syntax depends `govaluate`:
+
+* https://github.com/Knetic/govaluate
+
+Only boolean `true` for the result value is considered to be eligible.
+
+## Synopsis
+
+```yaml
+filter:
+  - type: cond
+    # (required) condition need to be satisfied
+    condition: "level == 'ERROR'"
+    # (required) filter config
+    filter:
+      - type: add_field
+        key: foo
+        value: bar
+```
+
+```yaml
+filter:
+  - type: cond
+    # (required) condition need to be satisfied
+    condition: "[nginx.access.url] ? [nginx.access.url] =~ '^/api/'"
```

Comment:
Sorry the the typo in README, it should no ternary check:
```yml
condition: "[nginx.access.url] =~ '^/api/'"
```
Just like those which in the PR description
Since the ternary operator of [govaluate](https://github.com/Knetic/govaluate) only accept `true` on left side:
https://github.com/Knetic/govaluate/blob/master/MANUAL.md#ternary-true-